src/mmw/js/src/modeling/models.js

```diff
@@ -342,6 +349,9 @@ var ScenarioModel = Backbone.Model.extend({
     // Poll the taskModel for results and reset the results collection when done.
     // If not successful, the results collection is reset to be empty.
     getResults: function() {
+        this.updateInputHash();
```

Comment:
Had to move these up so that `taskHelper` is initialized with the correct values. `updateInputHash` must execute before we assign `inputmod_hash` to `taskHelper`.